docs/developing.md

```diff
@@ -135,6 +135,31 @@ Alternatively, you can run the Dep command directly.
 dep ensure -v
 ```
 
+## Using Serving Certificates
+
+The hiveutil command includes a utility to generate Letsencrypt certificates for use with clusters you create in Hive.
+
+Prerequisites:
+* The `certbot` command must be available and in the path of your machine. You can obtain by following instructions at:
+  [https://certbot.eff.org/docs/install.html](https://certbot.eff.org/docs/install.html)
+* You must have credentials for AWS available in your command line, either by a configured ~/.aws/credentials or environment variables.
+
+### Generating a Certificate
+
+1. Ensure that the hiveutil binary is available (`make hiveutil`)
+2. Run: `hiveutil certificate create ${CLUSTER_NAME} --domain ${BASE_DOMAIN}` 
```

Comment:
Should we also provide a util for the user to delete the certificate that they created?

Reply:
There's really no corresponding delete. All the create does is that it places a new cert key in the output directory. If you don't want it anymore, just delete the file :)

Reply:
That makes sense. I wasn't sure if it created an artifact on AWS as well or not.

Reply:
It does create a DNS record on AWS but only temporarily to validate that you own the domain you’re getting a cert for.

Reply:
Looks like the example command needs be be fixed to `$ #bin/hiveutil certificate create ${CLUSTER_NAME} --base-domain ${BASE_DOMAIN}`
As the usage has `  --base-domain` instead of `--domain`
```
$ bin/hiveutil certificate create
Error: accepts 1 arg(s), received 0
Usage:
  hiveutil certificate create CLUSTER_NAME [flags]

Flags:
      --base-domain string   Base domain for the cluster (default "new-installer.openshift.com")
      --email string         email address to use for certificate (default "lmohanty@redhat.com")
  -h, --help                 help for create
      --output-dir string    Output directory for certs. Defaults to home directory (default "/home/lmohanty")
      --region string        AWS region for certificate hosted zone (default "us-east-1")
      --wait-time duration   amount of time to wait after creating dns entries so that they can be authenticated (default 30s)
```